plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -565,6 +565,19 @@ public void testPathColumn()
         }
     }
 
+    @Test
+    public void testTableLocationTrailingSpace()
+    {
+        String tableName = "table_with_space";
+        String tableLocationWithSpace = tableName + " ";
```

Comment:
I think you can inline `tableLocationWithSpace` OR construct fully qualified path here like `String tableLocationWithTrailingSpace = "s3://" + bucketName + "/" + tableName + " ";`